src/test/java/hw1/tests/CalculatorDivisionTest.java

```diff
@@ -0,0 +1,16 @@
+package hw1.tests;
+
+import hw1.common.AbstractBaseTest;
+import hw1.common.CalculatorDataProvider;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorDivisionTest extends AbstractBaseTest {
+    @Test(dataProviderClass = CalculatorDataProvider.class,
```

Comment:
Here should be empty line between class name and annotation. Please follow Java Code Convention

Reply:
fixed